docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -15,15 +15,33 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
 
+#### 2024-02-23
+
+Released: `@100mslive/hms-video-store@0.11.7`, `@100mslive/react-sdk@0.9.7`, `@100mslive/hls-player@0.2.7`, `@100mslive/roomkit-react@0.2.7`
+
+#### Added:
+
+-   Roomkit Prebuilt: Integrated Krisp for noise cancellation
+-   Roomkit Prebuilt: Support for room title and description which can be set from the dashboard customiser
+-   Roomkit Prebuilt: Chat notifications will be displayed while screensharing and the call tab is inactive
```

Comment:
This should be browser notifications, otherwise might be confusing

Reply:
"Chat notifications will be displayed by the browser while screensharing when the call tab is inactive"
Is this better?

Reply:
Browser notification will be displayed for chat while screensharing when the tab is inactive